components/camel-casper/src/main/java/org/apache/camel/component/casper/examples/DemoApp.java

```diff
@@ -0,0 +1,204 @@
+package org.apache.camel.component.casper.examples;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.casper.CasperConstants;
+import org.apache.camel.component.jackson.JacksonDataFormat;
+import org.apache.camel.impl.DefaultCamelContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.syntifi.casper.sdk.model.auction.AuctionState;
+import com.syntifi.casper.sdk.model.balance.BalanceData;
+import com.syntifi.casper.sdk.model.block.JsonBlock;
+import com.syntifi.casper.sdk.model.era.JsonEraValidators;
+import com.syntifi.casper.sdk.model.stateroothash.StateRootHashData;
+
+/**
+ * Demo Class loading Casper components example routes  
+ * @author p35862
+ *
+ */
+
+@SuppressWarnings("deprecation")
+public class DemoApp {
```

Comment:
As this PR is very big already, I'd consider moving this as a separate PR to the [camel-examples repository](https://github.com/apache/camel-examples)

Reply:
fixed